src/components/navBar/navBar.spec.js

```diff
@@ -156,6 +156,17 @@ describe('mdNavBar', function() {
 
       expect($scope.selectedTabRoute).toBe('tab2');
     });
+
+    it('does not update tabs if tab controller is undefined', function() {
+      $scope.selectedTabRoute = 'tab1';
+
+      spyOn(Object.getPrototypeOf(ctrl), '_updateInkBarStyles');
```

Comment:
It seems to actually be necessary for some reason. Just ran a few tests:
 `spyOn(ctrl, '_upateInkBarStyles');` results in the test failing with this error:  <toHaveBeenCalled> : Expected a spy, but got Function.
 `spyOn(ctrl.prototype, '_upateInkBarStyles');` also results in the test failing with this error:  <spyOn> : could not find an object to spy upon for _updateInkBarStyles()

The expect can be modified to utilize the ctrl object instead and the test passes as expected.

Any ideas?